src/app/components/candidate-card-popup/candidate-card-popup/candidate-card-popup.component.ts

```diff
@@ -1,110 +1,267 @@
-import { Component, Input, Output, EventEmitter, OnInit } from '@angular/core';
+import { Component, Input, Output, EventEmitter, OnInit, OnDestroy } from '@angular/core';
 import { MatSliderChange } from '@angular/material/slider';
-import axios from 'axios';
+import axios, { AxiosResponse } from 'axios';
```

Comment:
Mb it might be better to use our api service or angular httpclient.

Reply:
may be, but axios better for me